be/src/common/config.h

```diff
@@ -271,7 +274,7 @@ namespace config {
     // Port to start debug webserver on
     CONF_Int32(webserver_port, "8040");
     // Number of webserver workers
-    CONF_Int32(webserver_num_workers, "5");
+    CONF_Int32(webserver_num_workers, "48");
```

Comment:
48 is some big, how about 12?

Reply:
if http server only for normal  light http request, I think 12, or even 5 is ok. but stream load is a heavy http request， once user use stream load to insert data, more threads can reduce the risk of no reponse from http server. the time cost for switch between threads i think we can ignore when compared to time cost in stream load.   48 threads don't cost too much memory resources. Above all. The advantages outweigh the disadvantages.